proposals/simd/ImplementationStatus.md

```diff
@@ -1,150 +1,162 @@
+| `f64x2.min`                 | `-munimplemented-simd128` |                    | :heavy_check_mark: | :heavy_check_mark: |
+| `f64x2.max`                 | `-munimplemented-simd128` |                    | :heavy_check_mark: | :heavy_check_mark: |
+| `i32x4.trunc_s/f32x4:sat`   |               `-msimd128` | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| `i32x4.trunc_u/f32x4:sat`   |               `-msimd128` | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| `i64x2.trunc_s/f64x2:sat`   | `-munimplemented-simd128` |                    | :heavy_check_mark: | :heavy_check_mark: |
+| `i64x2.trunc_u/f64x2:sat`   | `-munimplemented-simd128` |                    | :heavy_check_mark: | :heavy_check_mark: |
+| `f32x4.convert_s/i32x4`     |               `-msimd128` | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| `f32x4.convert_u/i32x4`     |               `-msimd128` | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| `f64x2.convert_s/i64x2`     | `-munimplemented-simd128` |                    | :heavy_check_mark: | :heavy_check_mark: |
+| `f64x2.convert_u/i64x2`     | `-munimplemented-simd128` |                    | :heavy_check_mark: | :heavy_check_mark: |
+| `v8x16.swizzle`             |                           |                    | :heavy_check_mark: |                    |
+| `v8x16.shuffle`             |                           |                    | :heavy_check_mark: | :heavy_check_mark: |
+| `i16x8.convert_s/i8x16Low`  |                           |                    |                    |                    |
+| `i16x8.convert_s/i8x16High` |                           |                    |                    |                    |
+| `i16x8.convert_u/i8x16Low`  |                           |                    |                    |                    |
+| `i16x8.convert_u/i8x16High` |                           |                    |                    |                    |
+| `i32x4.convert_s/i16x8Low`  |                           |                    |                    |                    |
+| `i32x4.convert_s/i16x8High` |                           |                    |                    |                    |
+| `i32x4.convert_u/i16x8Low`  |                           |                    |                    |                    |
+| `i32x4.convert_u/i16x8High` |                           |                    |                    |                    |
+| `i16x8.convert_s/i32x4`     |                           |                    |                    |                    |
+| `i16x8.convert_u/i32x4`     |                           |                    |                    |                    |
+| `i8x16.convert_s/i8x16`     |                           |                    |                    |                    |
+| `i8x16.convert_u/i8x16`     |                           |                    |                    |                    |
```

Comment:
V8 implements some of these, so can the table reflect that? Also cc @tlively does the toolchain support this already?

Reply:
Done, i also found some typos and ordering issues, fixed in next commit